danceradvent/public/articles/2023/XX-dancer-liteblog.pod

```diff
@@ -0,0 +1,210 @@
+fine-tune your Liteblog site.
+There is no database involved, on-purpose.
+
+Before LiteBlog can work its magic, you need a Dancer2 app. Generate it:
+
+  $ dancer2 gen -a my_app
+
+Then, run the LiteBlog scaffolder from within the application directory:
+
+  $ cd my_app
+  $ liteblog-scaffold .
+
+This command populates your app directory with LiteBlog's assets, such as CSS
+files, views, and even starter content.
+
+To hook it all up, modify the main app module to enable the LiteBlog plugin:
+
+  package My::App;
+  use Dancer2;
+  use Dancer2::Plugin::LiteBlog;
+
+  liteblog_init;
+
+  1;
+
+Now, your Dancer2 app is a Liteblog. A default '/' route is defined, and widgets
+(that can be defined in your liteblog's settings) can also define their own
+routes.
+The blog engine is actually a Liteblog Widget. But let's just see what happens
+right there, when Liteblog is invoked from a pure Dancer2 app.
+
+Just launch your app and open its URL with your browser.
+You'll be greeted with a splash screen:
+
+  $ plackup bin/app.psgi
+
+Follow the on-screen instructions to enable features and widgets by adding the
+provided config snippet to your 'config.yml', then restart.
+
+Now, you have C<Activities> cards enabled on the home and a nice article that you
+can read thanks to the C<Blog> widget. 
+
+=head2 Publishing content
+
+LiteBlog makes content creation intuitive. Articles reside in a specified root
+directory as markdown files with coresponding 'meta.yml' for metadata. To
+publish, go in your C<articles/> folder (it has been created by the scaffolder),
+and then follow theses steps:
+
+Create a directory for the category of your post (if not, the article will be
+considered a page). Note that the directory name is the post slug.
+
+Inside, you first need to create the metadata file: C<meta.yml>. It is mandatory
+and should at least contain a title. Here is an example of a valid metadata file
+for an article:
+
+    ---
+    title: "Some Great LiteBlog Article"
+    image: "featured.jpg"
+    tags:
+      - "perl"
+      - "dancer"
+
+Finally, you can now create the C<content.md> file, which, as its name suggests,
+should be written in Markdown format and will be used as the actual content of
+the article.
+
+The easiest way to get started is to edit the scaffolded article provided
+once you bootstrap your Liteblog app.
+
+=head2 Main features of a Liteblog site
+
+As of version 0.05, LiteBlog's toolbox includes:
+
+=over 4
+
+=item B<Activities cards>: Display your profiles and activities in an elegant grid.
+
+=item B<Blog engine>: Effortlessly manage and publish articles.
+
+=item B<Caching system>: Speed up page rendering without breaking a sweat.
+
+=item B<Responsive design>: Your blog will look great on any device.
+
+=item B<Highlight.js support>: Syntax highlighting for articles featuring code.
+
+=back
+
+=head2 Deploying as a Docker image
+
+It's quite classic these days to go with a Docker image for deployment. 
```

Comment:
Very minor nit: I'd use `typical` instead of `classic`. 
```suggestion
It's quite typical these days to go with a Docker image for deployment. 
```

Reply:
Sounds better indeed! Thanks!